docs/getting-started/tutorials/auth-architecture/introduction.md

```diff
@@ -0,0 +1,20 @@
+---
+id: intro 
+title: Auth Architecture Introduction
+sidebar_label: Introduction
+---
+
+Welcome to the Mia-Platform AuthN/AuthZ Architecture tutorial! This is an holistic guide on the crucial realm of authentication, exploring all the possible AuthN/AuthZ architectures you could implement to secure your Digital Platform built with Mia-Platform.
+
+We'll start with the basics, addressing the simplest scenarios first, and gradually advance, tackling multi-project architectures and integrating multiple external Identity Providers. 
+
+For each of the following scenarios, we will provide an overview of the architecture and a complete step-by-step tutorial to implement it.
+
+### Scenarios
```

Comment:
Would be nice to explain which scenario is better for each situation. In that way we can guide the developer to the correct solution.